working-groups/rfc-2229/NOTES.md

```diff
@@ -1,5 +1,14 @@
 # Meeting Notes
-Video recording of some of our earlier meetings
-- https://www.youtube.com/watch?v=fMopdkn5-Xw
-- https://www.youtube.com/watch?v=pLmVhSB-z4s
-- https://www.youtube.com/watch?v=2QCuNtISoYc
+
+## Roadmap
+The project roadmap can be found [here][roadmap].
+
+## Meetings
```

Comment:
I think we should move this information to the README somehow (we probably want to edit the template, too). I also think we should make a meetings directory and try to add a few notes after each meeting, maybe -- in this case probably just a link to the video and a few bullets about what it was about (e.g., a file `minutes/2019.03.05-roadmap-plan.md` or something).

Reply:
So in that case should we delete the NOTES.md file ?